lightrag/api/routers/document_routes.py

```
@@ -11,14 +11,17 @@
 from pathlib import Path
 from typing import Dict, List, Optional, Any, Literal
-from fastapi import APIRouter, BackgroundTasks, Depends, File, HTTPException, UploadFile
+from fastapi import APIRouter, BackgroundTasks, Depends, File, HTTPException, UploadFile, Request
 from pydantic import BaseModel, Field, field_validator
 
 from lightrag import LightRAG
 from lightrag.base import DocProcessingStatus, DocStatus
 from lightrag.api.utils_api import get_combined_auth_dependency
 from ..config import global_args
 
+from ..utils_api import extract_user_id
```

Ruff (0.8.2)

`..utils_api.extract_user_id` imported but unused

Remove unused import: `..utils_api.extract_user_id`

(F401)